src/en/hiveworks/src/eu/kanade/tachiyomi/extension/en/hiveworks/Hiveworks.kt

```diff
@@ -178,6 +201,7 @@ class Hiveworks : ParsedHttpSource() {
     // Included to call custom error codes
     override fun fetchChapterList(manga: SManga): Observable<List<SChapter>> {
         return if (manga.status != SManga.LICENSED) {
+            val uri = Uri.parse(manga.url).buildUpon()
```

Comment:
unused?
tho upon looking again, this method override isn't changing anything from the implementation in super class, so should remove it.

Reply:
Removed the accidental leftover variable. However, I think the method override is still needed, it adds the conditional that reports an error when the manga is licensed (some Hiveworks series listed in the index are available on web while others are not).